test/module/irohad/execution/command_validate_execute_test.cpp

```diff
@@ -1449,14 +1392,10 @@ TEST_F(TransferAssetTest, InvalidWhenNoPermissions) {
  */
 TEST_F(TransferAssetTest, InvalidWhenNoDestAccount) {
   // TODO 2018-04-20 Alexey Chernyshov - IR-1276 - rework with CommandBuilder
-  command = clone(
-      *(TestTransactionBuilder()
-            .transferAsset(kAdminId, "noacc", kAssetId, kDescription, "150.00")
-            .build()
-            .commands()
-            .front()));
+  command = buildCommand(TestTransactionBuilder().transferAsset(
+      kAdminId, "noacc", kAssetId, kDescription, "150.00"));
```

Comment:
Again overflow, constants may significantly reduce such errors
Add please: `kMax` and `kOverflow`